Sources/Mantis/CropViewController/CropViewController.swift

```diff
@@ -378,9 +420,42 @@ private extension CropViewController {
 extension CropViewController: CropViewDelegate {    
     func cropViewDidBecomeResettable(_ cropView: CropViewProtocol) {
         cropToolbar.handleCropViewDidBecomeResettable()
-        delegate?.cropViewControllerDidImageTransformed(self)
         delegate?.cropViewControllerDidImageTransformed(self, transformation: cropView.makeTransformation())
```

Comment:
Since you removed `delegate?.cropViewControllerDidImageTransformed(self)` here, can you also help remove `func cropViewControllerDidImageTransformed(_ cropViewController: CropViewController)` from CropViewControllerDelegate. Then we can just stop supporting this deprecated delegate function.

Reply:
removed

Reply:
Thank you!